test/dns/output/dns.abi.json

```diff
@@ -0,0 +1,187 @@
+                    "multi_result": true
+                }
+            ]
+        },
+        {
+            "name": "resolveFromHash",
+            "inputs": [
+                {
+                    "name": "name_hash",
+                    "type": "H256"
+                }
+            ],
+            "outputs": [
+                {
+                    "type": "OptionalResult<Address>",
+                    "multi_result": true
+                }
+            ]
+        },
+        {
+            "name": "checkPending",
+            "inputs": [
+                {
+                    "name": "name",
+                    "type": "bytes"
+                }
+            ],
+            "outputs": [
+                {
+                    "type": "OptionalResult<Address>",
+                    "multi_result": true
+                }
+            ]
+        },
+        {
+            "name": "resetPending",
+            "inputs": [
+                {
+                    "name": "name",
+                    "type": "bytes"
+                }
+            ],
+            "outputs": []
+        },
+        {
+            "name": "claim",
+            "inputs": [],
+            "outputs": []
+        },
+        {
+            "name": "getRegistrationCost",
+            "inputs": [],
+            "outputs": [
+                {
+                    "type": "BigUint"
+                }
+            ]
+        },
+        {
+            "name": "getContractOwner",
+            "inputs": [],
+            "outputs": [
+                {
+                    "type": "Address"
+                }
+            ]
+        },
+        {
+            "name": "getOwnShardId",
+            "inputs": [],
+            "outputs": [
+                {
+                    "type": "u8"
+                }
+            ]
+        },
+        {
+            "name": "nameHash",
+            "inputs": [
+                {
+                    "name": "name",
+                    "type": "bytes"
+                }
+            ],
+            "outputs": [
+                {
+                    "type": "H256"
+                }
+            ]
+        },
+        {
+            "name": "nameShard",
+            "inputs": [
+                {
+                    "name": "name",
+                    "type": "bytes"
+                }
+            ],
+            "outputs": [
+                {
+                    "type": "u8"
+                }
+            ]
+        },
+        {
+            "name": "validateName",
+            "inputs": [
+                {
+                    "name": "name",
+                    "type": "bytes"
+                }
+            ],
+            "outputs": []
+        },
+        {
+            "name": "version",
+            "inputs": [],
+            "outputs": [
+                {
+                    "type": "bytes"
+                }
+            ]
+        }
+    ],
+    "types": {}
+}
```

Comment:
Newline at EOF.

Reply:
hmm, this is auto-generated so the fix is somewhere else
probably no need for ABI here